subsys/bluetooth/mesh/lightness_srv.c

```diff
@@ -178,7 +178,8 @@ static void handle_linear_get(struct bt_mesh_model *model,
 void lightness_srv_change_lvl(struct bt_mesh_lightness_srv *srv,
 			      struct bt_mesh_msg_ctx *ctx,
 			      struct bt_mesh_lightness_set *set,
-			      struct bt_mesh_lightness_status *status)
+			      struct bt_mesh_lightness_status *status,
+			      bool disable_ctrl)
```

Comment:
IMO it makes more sense to make this a separate function

Reply:
Hm. More than half of the time you call one you'd have to call both. And, apparently, as this bugfix was needed, that's easy to forget. That was my reasoning, anyways. But I'm OK with splitting it again if you still feel it's better :)

Reply:
IMO it's better, and will let us disengage control for other reasons as well

Reply:
Split out now